freqtrade/data/btanalysis.py

```diff
@@ -129,12 +129,16 @@ def load_trades_from_db(db_url: str) -> pd.DataFrame:
     return trades
 
 
-def load_trades(source: str, db_url: str, exportfilename: str) -> pd.DataFrame:
+def load_trades(source: str, db_url: str, exportfilename: str, skip_trades: bool) -> pd.DataFrame:
     """
     Based on configuration option "trade_source":
     * loads data from DB (using `db_url`)
     * loads data from backtestfile (using `exportfilename`)
     """
+    if skip_trades:
```

Comment:
I would call the parameter to this function either "empty" or "no_trades" (irrelevant of the cli argument naming) - assuming that this function is used by other means too (interactive analysis, for example) that naming will give the parameter a clearer meaning.